### ISSUE TITLE
angular-recaptcha requires jQuery

### ISSUE THREAD

**Issue by DSpeichert** [VividCortex/angular-recaptcha#178]
Errors occur when Angular is used without jQuery (then Angular uses jqLite internally), e.g. at lines:
```js
// Generate link on demand
var script = $document.get(0).createElement('script');
script.async = true;
script.defer = true;
script.src = 'https://www.google.com/recaptcha/api.js?onload='+provider.onLoadFunctionName+'&render=explicit';
$document.get(0).body.appendChild(script);
```

as `$document` does not have method `get()`.

Introduced in 82d33e6adabdd81a0341b6e68866f28b0f760522

Fix provided in #181

**Comment by xfakt-pj** [comment]
watching this, let me know if any fixes need real-life testing.

**Comment by DSpeichert** [comment]
A new fix proposed in #181.
